src/OrbitGl/TimelineUiTest.cpp

```diff
@@ -88,6 +89,17 @@ class TimelineUiTest : public TimelineUi {
                                                          GlCanvas::kZValueTimeBarLabel));
     EXPECT_TRUE(mock_batcher_.IsEverythingBetweenZLayers(GlCanvas::kZValueTimeBar,
                                                          GlCanvas::kZValueTimeBarLabel));
+
+    // The label is the only thing that can be out of the expected space for the timeline.
+    const char* kOneMonthLabel = "730:00:00.000000000";
+    const float kMaxSizeLabel =
```

Comment:
Should we call this just `label_width`?  

This is a width, rather than a label. Also, I wonder why this is the "maximum size".

Reply:
We are only supporting OneMonthTimestamps in the Timeline, so this is the biggest value and also the longest possible label.

Reply:
Ah, thanks for the explanation. Then maybe `kMaxLabelWidth`